config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-periodics-master.yaml

```diff
@@ -2,18 +2,22 @@ periodics:
 - interval: 6h
   name: periodic-controller-runtime-test-master
   decorate: true
+  labels:
+      preset-service-account: "true"
   extra_refs:
   - org: kubernetes-sigs
     repo: controller-runtime
     base_ref: master
   spec:
     containers:
-    - image: golang:1.13
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200720-4766100-1.18
       command:
+      - runner.sh
       - ./hack/ci-check-everything.sh
       resources:
         requests:
-          cpu: 6000m
+          cpu: "6000m"
```

Comment:
The nodes have 7400M allocatable and 16 gig ram, I am reasonable sure this won't make a difference because we get an exclusive node anyways: https://kubernetes.slack.com/archives/C09QZ4DQB/p1594929742176700?thread_ts=1594858427.174300&cid=C09QZ4DQB

Reply:
That's interesting, Should we try to allocate ~7 then to make sure nothing else is scheduled? AFAIK, there is lots of jobs that don't have explicit CPU/memory requests that could still get scheduled, unless there is a higher default sometwhere

Reply:
Yeah from what I can see we should be requesting 7400m to guarantee exlusivity: a81e294b95401dddeb1c8fea5514f8a6b1ef11af

Reply:
Placed it to 7000m to have some headroom, happy to bump to 7300 or 7400